functions/FunctionsExample/FunctionsExample/View/CapitalizeMessageView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  Copyright 2022 Google LLC
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+import Firebase
+
+struct CapitalizeMessageView: View {
+  @State private var comment: String = ""
+  @State private var outcome: String = ""
+  private var functions = Functions.functions()
+  var body: some View {
+    BackgroundFrame(
+      title: "CapitalizeMessage",
+      description: "Capitalize the input message and return it.",
+      buttonAction: didTapAddMessage
+    ) {
+      VStack {
+        TextField("", text: $comment, prompt: Text("Type message"))
+          .background(RoundedRectangle(cornerRadius: 16).fill(Color(.systemGray5)))
+          .padding()
+        Text(outcome)
+      }
+      .padding()
+    }
+  }
+
+  func didTapAddMessage() {
+    Task {
+      do {
+        let result = try await functions.httpsCallable("capitalizeMessage")
+          .call(["text": $comment.wrappedValue])
+        if let data = result.data as? [String: Any], let text = data["text"] as? String {
+          self.outcome = text
+        }
```

Comment:
Should we use `FirebaseFunctionsSwift` to showcase the recent Codable-related changes? This code could be further simplified like:
https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseFunctionsSwift/Tests/IntegrationTests.swift#L156-L163

Imo, I think we should use the `FirebaseFunctionsSwift` extension, mainly to showcase Swift API improvements as they are released. cc: @morganchen12

Reply:
Good point, just add it and the new function is neat.